examples/bootstrap.php

```diff
@@ -17,7 +17,7 @@
 use Symfony\Component\HttpClient\HttpClient;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
 
-require_once __DIR__.'/vendor/autoload.php';
+require_once __DIR__.'/../src/platform/vendor/autoload.php';
```

Comment:
please revert this. you can execute the examples with you local changes like this:

```
cd examples
composer install
../link
```

this creates symlinks to your local versions of the packages in the vendor folder of the examples

Reply:
Oh! Committed by mistake.
I tried linking the vendor folder to the local changes, but it didn't work for some reason.
I kept getting `SomeClass` not found exception, so I did this as a temporary measure to run the examples.